### PR TITLE
Add json file for descriptive items for a few eateries

### PR DESCRIPTION
**Pull request by orkosinha**
For Atrium, Bus Stop Bagels, and Nasties

### CODE REVIEW

File: DiningStack/expandedItems.json [hunk 1]
```diff
@@ -0,0 +1,1041 @@
...
+                        "hummus"
+                      ]
+                    },
+                    {
+                      "label": "Cheese",
+                      "options": [
+                        "American",
+                        "Swiss",
+                        "Brie",
+                        "provolone",
```
Comment (ilikesymmetry) [inline DiningStack/expandedItems.json]: keep capitalization consistent

File: DiningStack/expandedItems.json [hunk 2]
```diff
@@ -0,0 +1,1041 @@
...
+            {
+              "station": "Custom Deli Sandwiches",
+              "diningItems": [
+                {
+                  "item": "Gourmet Sandwiches",
+                  "category": "",
+                  "options": [
+                    {
+                      "label": "Bread",
+                      "options": [
```
Comment (ilikesymmetry) [inline DiningStack/expandedItems.json:84]: might not be ideal to have two fields named `options`, we should rename one

**Review by ilikesymmetry — COMMENTED** [review]
Great work dude! I have a couple of notes for things that might be worth changing.  The first is the `options` field.  If we use python's built in `.get('options', [])` we can get away with omitting the field entirely for items with no options which could slim down file bloat.  Another potential direction we could go is if a station has the same price for all its items, having a `stationPrice` field with that price so that not every item needs to write the same value.  When writing the parsing, we could also have a `stationPrice` if there are many repeats, but allow each item to have its own price if we want to override this `stationPrice`.  The same principle could also be possible with a station with the same options (drink sizes for example). The main things I'm looking at are removing file bloat here.  With that said, I'm not sure if those things would be worth pursuing because they may complicate the file out of necessity and add too much nuance. @meganle what do you think?

File: DiningStack/expandedItems.json [hunk 3]
```diff
@@ -0,0 +1,1041 @@
...
+                  "options": [],
+                  "price": "$2.49"
+                },
+                {
+                  "item": "Hot Cocoa",
+                  "options": [],
+                  "price": "$2.89"
+                },
+                {
+                  "item": "Frppuccino",
```
Comment (meganle) [inline DiningStack/expandedItems.json]: spelling

File: DiningStack/expandedItems.json (hunk 2 above)
Comment (orkosinha) [reply DiningStack/expandedItems.json:84]: Renamed

File: DiningStack/expandedItems.json [hunk 4]
```diff
@@ -12,22 +12,22 @@
...
                 {
                   "item": "Cheese Pizza",
-                  "options": [],
+                  "choices": [],
                   "price": "$4.49"
                 },
                 {
                   "item": "Veggie Pizza",
-                  "options": [],
+                  "custom": [],
```
Comment (ilikesymmetry) [inline DiningStack/expandedItems.json]: lol you mean `choices`

**Review by ilikesymmetry — APPROVED** [review]
change that one `custom` to `choices` and you're chillin

**Review by young-k — APPROVED** [review]
#shipit

**Pull request merged**